pkg/k8sutil/k8sutil.go

```diff
@@ -203,10 +203,9 @@ func IsAllowed(
 						resource += "/" + ra.Name
 					}
 
-					switch {
-					case ns == v1.NamespaceAll:
+					if ns == v1.NamespaceAll {
```

Comment:
we could have this instead (which I prefer personally)

````
switch ns {
    case v1.NamespaceAll:
        ....
    default:
        ....
}
````

Reply:
Thanks @simonpasquier 
Fixed